dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -75,6 +75,8 @@ isc-dhcp-relay-{{ vlan_name }}
 [program:isc-dhcp-relay-{{ vlan_name }}]
 {# We treat this VLAN as a downstream interface (-id), as we only want to listen for requests #}
 command=/usr/sbin/dhcrelay -d -m discard -a %%h:%%p %%P --name-alias-map-file /tmp/port-name-alias-map.txt -id {{ vlan_name }}
+{#- Dual ToR Option #}
+{% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %} -U Loopback0 -dt{% endif -%}
```

Comment:
Should `Loopback0` be hardcoded, or should we retrieve the name from the DB?

Reply:
`Loopback0` is a key in `LOOPBACK_INTERFACE`
```
    "LOOPBACK_INTERFACE": {
        "Loopback1|2001:506:28:500::1/128": {}, 
        "Loopback1|10.20.8.199/32": {}, 
        "Loopback2|10.212.64.0/32": {}, 
        "Loopback2|2603:10e1:100:d::1/128": {}, 
        "Loopback0|2a01:111:e210:50::/128": {}, 
        "Loopback0": {}, 
        "Loopback1": {}, 
        "Loopback2": {}, 
        "Loopback0|10.152.0.40/32": {}
    }, 
```

Reply:
we have to hardcode it because Loopback0 is a (part of) key of the LOOPBACK_INTERFACE table and there are multiple keys in this table